packages/forklift-console-plugin/src/modules/Plans/utils/types/PlanEditAction.ts

```diff
@@ -0,0 +1 @@
+export type PlanEditAction = 'PLAN' | 'VMS';
```

Comment:
it will allow us next (with small changes [like this](https://github.com/kubev2v/forklift-console-plugin/blob/cee5f94c951296187612cb7653c6d22cee49a8dc/packages/forklift-console-plugin/src/modules/Plans/actions/PlanActionsDropdownItems.tsx)) to edit plans too